docs/docs-content/clusters/edge/edgeforge-workflow/palette-canvos/build-content-bundle.md

```diff
@@ -295,7 +297,7 @@ Creating a content bundle provides several benefits that may address common use
      --cred-file-path <FILE_PATH> \
      --include-palette-content \
      --cluster-definition-name <CLUSTER_DEFINITION_FILENAME> \
-     --cluster-definition-profile-ids <CLUSTER_PROFILE_IDS>
+     --cluster-definition-profile-ids <CLUSTER_PROFILE_IDS> |
```

Comment:
@lennessyy did you mean to insert a pipe sign here?

Reply:
nope - thanks for catching that